From: Tomas Henzl <thenzl@redhat.com>
Date: Sun, 29 Aug 2010 15:49:56 -0400
Subject: [block] cciss: fix raid level sysfs permissions
Message-id: <1283097002-3341-58-git-send-email-thenzl@redhat.com>
Patchwork-id: 27898
O-Subject: [RHEL6 PATCH 57/63] cciss: fix raid level sysfs permissions
Bugzilla: 568830
RH-Acked-by: Neil Horman <nhorman@redhat.com>

Fix raid_level /sys attribute permissions.  It should not be writable.

diff --git a/drivers/block/cciss.c b/drivers/block/cciss.c
index 49cd634..c578efa 100644
--- a/drivers/block/cciss.c
+++ b/drivers/block/cciss.c
@@ -698,7 +698,7 @@ static ssize_t cciss_show_raid_level(struct device *dev,
 	return snprintf(buf, strlen(raid_label[raid]) + 7, "RAID %s\n",
 			raid_label[raid]);
 }
-static DEVICE_ATTR(raid_level, S_IRUGO | S_IWUSR, cciss_show_raid_level, NULL);
+static DEVICE_ATTR(raid_level, S_IRUGO, cciss_show_raid_level, NULL);
 
 static ssize_t cciss_show_usage_count(struct device *dev,
 				      struct device_attribute *attr, char *buf)
